app/pods/docs/validators-reference/required-unless-data/template.hbs

```diff
@@ -11,3 +11,34 @@
 {{#code-sample lang="js"}}
 firstname: 'requiredUnlessData(foo)'
 {{/code-sample}}
+
+<h3>Example:</h3>
+{{#code-sample lang="js"}}
+// would throw error only if foo key in the
+//data hash would not have a truthy value
+firstName: 'requiredUnlessData(foo)'
+{{/code-sample}}
+
+<p>It would however mark the field as valid if we passed a following <code>data</code> object to our <code>lf-form</code> component:</p>
+
+{{#code-sample lang="js" title="components/some-components/component.js"}}
+import Ember from 'ember';
+const { Component } = Ember;
+export default Component.extend({
+  data: {
+    foo: true
+  },
+
+  rules: {
+    firstName: 'requiredUnlessData(foo)'
+  }
+});
+{{/code-sample}}
+
+<br>
```

Comment:
Purely presentational, to add some margin between two code samples with titles.

I took a quick look into existing CSS, seems more than one-liner/5 min. fix to do it properly. Happy to do separate PR though.